### PR TITLE
add copyright

### PR DESCRIPTION
**Pull request by jya-kmu**
Add copyright to hermes.

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Copyright Notice and License Terms for
+Hermes I/O Buffering Software Library and Utilities
+
+------------------------------------------------------------------------------
+
+Hermes I/O Buffering Software Library and Utilities
+Copyright 2018-20201, The HDF Group and Illinois Institute of Technology
```
Comment (ChristopherHogan) [inline COPYING:7]: 20201 should be 2021.

Comment (jya-kmu) [reply COPYING:7]: I will fix that...

File: COPYING [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+available either publicly, or directly to the copyright holders, without
+imposing a separate written license agreement for such Enhancements, then you
+hereby grant the following license: a non-exclusive, royalty-free perpetual
+license to install, use, modify, prepare derivative works, incorporate into
+other computer software, distribute, and sublicense such enhancements or
+derivative works thereof, in binary and source code form.
+
+------------------------------------------------------------------------------
+
+Hermes were developed with support from the National Science Foundation (NSF)
```
Comment (ChristopherHogan) [inline COPYING:51]: "Hermes was"

**Comment by jya-kmu** [comment]
Reread it a couple of times. Hope it is good to go.

**Review by ChristopherHogan — APPROVED** [review]

**Pull request merged**